src/Microsoft.AspNet.Mvc.Razor/Logging/RazorViewEngineLoggerExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public static class RazorViewEngineLoggerExtensions
```

Comment:
I thought we follow the pattern of using the name of the file where logging happens.